modules/mono/glue/GodotSharp/Godot.SourceGenerators.Internal/ExtensionMethods.cs

```diff
@@ -94,22 +94,32 @@ public static string GetDeclarationKeyword(this INamedTypeSymbol namedTypeSymbol
         };
     }
 
-    private static SymbolDisplayFormat FullyQualifiedFormatOmitGlobal { get; } =
-        SymbolDisplayFormat.FullyQualifiedFormat
-            .WithGlobalNamespaceStyle(SymbolDisplayGlobalNamespaceStyle.Omitted);
-
-    public static string FullQualifiedName(this ITypeSymbol symbol)
-        => symbol.ToDisplayString(NullableFlowState.NotNull, FullyQualifiedFormatOmitGlobal);
-
     public static string NameWithTypeParameters(this INamedTypeSymbol symbol)
     {
         return symbol.IsGenericType ?
             string.Concat(symbol.Name, "<", string.Join(", ", symbol.TypeParameters), ">") :
             symbol.Name;
     }
 
-    public static string FullQualifiedName(this INamespaceSymbol symbol)
-        => symbol.ToDisplayString(FullyQualifiedFormatOmitGlobal);
+    private static SymbolDisplayFormat FullyQualifiedFormatOmitGlobal { get; } =
+        SymbolDisplayFormat.FullyQualifiedFormat
+            .WithGlobalNamespaceStyle(SymbolDisplayGlobalNamespaceStyle.Omitted);
+
+    private static SymbolDisplayFormat FullyQualifiedFormatIncludeGlobal { get; } =
+        SymbolDisplayFormat.FullyQualifiedFormat
+            .WithGlobalNamespaceStyle(SymbolDisplayGlobalNamespaceStyle.Included);
+
+    public static string FullQualifiedNameOmitGlobal(this ITypeSymbol symbol)
+        => symbol.ToDisplayString(NullableFlowState.NotNull, FullyQualifiedFormatOmitGlobal);
+
+    public static string FullQualifiedNameOmitGlobal(this INamespaceSymbol namespaceSymbol)
+        => namespaceSymbol.ToDisplayString(FullyQualifiedFormatOmitGlobal);
+
+    public static string FullQualifiedNameIncludeGlobal(this ITypeSymbol symbol)
+        => symbol.ToDisplayString(NullableFlowState.NotNull, FullyQualifiedFormatIncludeGlobal);
+
+    public static string FullQualifiedNameIncludeGlobal(this INamespaceSymbol namespaceSymbol)
+        => namespaceSymbol.ToDisplayString(FullyQualifiedFormatIncludeGlobal);
```

Comment:
ditto

Reply:
The overloads already existed before. Do you think it'd be better if I renamed the ones that take `ITypeSymbol` to something like: `FullQualifiedNameNotNullOmitGlobal`? It's a lot more verbose but maybe it's better to be explicit. Or maybe at this point we should just use `ToDisplayString` directly.

Reply:
Oops, never mind. There's no overload for `ISymbol` here, which is the one that I don't like. `ITypeSymbol` and `INamespaceSymbol` don't overlap.